### PR TITLE
Localizations in Chinese

### PR DESCRIPTION
**Pull request by cloudyyoung**
- Translated most of the newly-added untranslated strings
 - Optimized some expressions to be more native & natural
 - Uniformed terminolgies

### CODE REVIEW

**Comment by penguin0616** [comment]
It looks like GitHub never notified me about this pull request. Unfortunate.
There have been enough changes since your pull request that I'll have to insert your translations into them myself to avoid the conflict hassles. 

Additionally, how do you want to be credited? A link will be put on the workshop page, as well as in the language file. 
Do you have a steam page I can use, or do you want to be credited to your GitHub page?

File: scripts/language/chinese.lua [hunk 1]
```diff
@@ -579,49 +579,49 @@ return {
...
-	wereness_remaining = "木头值：%s / %s",
+	wereness_remaining = "木头值: %s / %s",
 
 	-- wintersfeasttable.lua
 
 	-- wintertreegiftable.lua
 	wintertreegiftable = {
-		ready = "You are <color=#bbffbb>eligible</color> for <color=#DED15E>rare gifts</color>.",
-		not_ready = "You must <color=#ffbbbb>wait %s more day(s)</color> before you can get another <color=#DED15E>rare gift</color>.",
+		ready = "你已可以打开 <color=#DED15E>礼物礼物</color>.",
```
Comment (cloudyyoung) [inline scripts/language/chinese.lua:601]: Correction: 
`ready = "你已可以打开 <color=#DED15E>稀有礼物</color>.",`

**Comment by cloudyyoung** [comment]
Oh sure, and my Steam page is: https://steamcommunity.com/id/cloudyyoung/. :)

**Comment by penguin0616** [comment]
Added by commit 3597a398d7afba5d6ac5e9ef4426978729c3013e

Credit to you for your translation efforts are in the file and Steam description. Thanks for taking the time to do this.

**Pull request closed without merging**